src/radium-stylesheet.js

```diff
@@ -8,6 +8,20 @@ import theme from "./radium-theme";
 // MozOsxFontSmoothing: "grayscale",
 // WebkitFontSmoothing: "antialiased"
 
+const codeStyles = {
+  fontFamily: theme.code,
+  fontSize: ".9em",
+  lineHeight: 1.375,
+  direction: "ltr",
+  textAlign: "left",
+  whiteSpace: "pre",
+  wordSpacing: "normal",
+  wordBreak: "normal",
+  tabSize: 4,
```

Comment:
yeah! Assuming that the space in question is indeed a tab! https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size